### DISCUSSION TITLE
Bookmark tagging

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#764] [Ideas]
### What feature would you like to see?

I would like to use the same bookmark tagging mechanism as in Firefox. My usual workflow for saving bookmarks involves pressing Ctrl+D, then pressing Tab twice to add tags. Unfortunately, I haven't been able to figure out how to enable this feature.

 
 ~ _Created automatically from issue: https://github.com/zen-browser/desktop/issues/687_

**Comment by RuthAja** [comment]
Hi, sorry if my English is bad.

actually this bookmark tag is very helpful for someone like me who has many types of bookmarks that I want to save and search for bookmarks based on tags like in firefox. It would be great if Zen could also create a tag mechanism like in firefox.

**Comment by livviest** [comment]
I encountered this today when I was creating a fresh profile and I noticed that I couldn't do this on the fresh profile when I had been on my main profile. 

Long story short: If none of your bookmarks have tags already, the default Firefox behavior is to just not show the tag field when adding a new bookmark. You can easily add tags to any bookmark from the bookmark manager window (Ctrl+Shift+O) and you should start seeing the tag field again. 

Hope this helps anyone stumbling across this in the future!

**Comment by riedel** [comment]
There used to be a great firefox extension to use tags more efficiently: https://github.com/wagle/tagspace-archive
It seized to work because the extension API is not exposed anymore. Better tagging support IMHO would be a great addition to Zen in general.
